packages/create-sitecore-jss/src/templates/nextjs-sxa/scripts/scaffold-component.ts

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
I'm not sure that I can see any difference of this file and nextjs/scripts/scaffold-component.ts that we have. Essentially this file is not needed